docs/content/_docs/query_language.html

```diff
@@ -227,72 +192,104 @@ <h4>Operators</h4>
+    <td><pre><code class="language-json">["or", ["=", &lt;foo&gt;, &lt;bar&gt;], ["=", &lt;foo&gt;, &lt;baz&gt;]]</code></pre></td>
   </tr>
 
   <tr>
-    <th scope="row">Tag Does Not Start With</th>
-    <td><pre><code class="language-hql">&lt;a&gt; !^ &lt;b&gt;</code></pre></td>
-    <td><pre><code class="language-json">["not", ["^", &lt;a&gt;, &lt;b&gt;]]</code></pre></td>
+    <th scope="row">Tag Not In</th>
+    <td>Tag foo does not equal to either bar or baz.</td>
+    <td><pre><code class="language-hql">&lt;foo&gt; not in [&lt;bar&gt;, &lt;baz&gt;]</code></pre></td>
+    <td><pre><code class="language-json">["not", ["or", ["=", &lt;foo&gt;, &lt;bar&gt;], ["=", &lt;foo&gt;, &lt;baz&gt;]]]</code></pre></td>
   </tr>
 
   <tr>
-    <th scope="row">Key Matches</th>
-    <td><pre><code class="language-hql">$key = &lt;a&gt;</code></pre></td>
-    <td><pre><code class="language-json">["key", &lt;a&gt;]</code></pre></td>
+    <th scope="row">Tag Exists</th>
+    <td>Tag foo exists.</td>
+    <td><pre><code class="language-hql">+&lt;foo&gt;</code></pre></td>
+    <td><pre><code class="language-json">["+", &lt;foo&gt;]</code></pre></td>
   </tr>
 
   <tr>
-    <th scope="row">Key Does Not Match</th>
-    <td><pre><code class="language-hql">$key != &lt;a&gt;</code></pre></td>
-    <td><pre><code class="language-json">["not", ["key", &lt;a&gt;]]</code></pre></td>
+    <th scope="row">Tag Does Not Exists</th>
+    <td>Tag foo does not exist.</td>
+    <td><pre><code class="language-hql">!+&lt;foo&gt;</code></pre></td>
+    <td><pre><code class="language-json">["not", ["+", &lt;foo&gt;]]</code></pre></td>
   </tr>
 
   <tr>
-    <th scope="row">Tag Exists</th>
-    <td><pre><code class="language-hql">+&lt;a&gt;</code></pre></td>
-    <td><pre><code class="language-json">["+", &lt;a&gt;]</code></pre></td>
+    <th scope="row">Tag Prefixed With</th>
+    <td>Tag foo is prefixed with bar.</td>
+    <td><pre><code class="language-hql">&lt;foo&gt; ^ &lt;bar&gt;</code></pre></td>
+    <td><pre><code class="language-json">["^", &lt;foo&gt;, &lt;bar&gt;]</code></pre></td>
   </tr>
 
   <tr>
-    <th scope="row">Tag In</th>
-    <td><pre><code class="language-hql">&lt;a&gt; in [&lt;b&gt;, ..]</code></pre></td>
-    <td><pre><code class="language-json">["or", ["=", &lt;a&gt;, &lt;b&gt;], ..]</code></pre></td>
+    <th scope="row">Tag Not Prefixed With</th>
+    <td>Tag foo is not prefixed with bar.</td>
+    <td><pre><code class="language-hql">&lt;foo&gt; !^ &lt;bar&gt;</code></pre></td>
+    <td><pre><code class="language-json">["not", ["^", &lt;foo&gt;, &lt;bar&gt;]]</code></pre></td>
   </tr>
 
   <tr>
-    <th scope="row">Tag Not In</th>
-    <td><pre><code class="language-hql">&lt;a&gt; not in [&lt;b&gt;, ..]</code></pre></td>
-    <td><pre><code class="language-json">["not", ["or", ["=", &lt;a&gt;, &lt;b&gt;], ..]]</code></pre></td>
+    <th scope="row">True</th>
+    <td><pre><code class="language-hql">true</code></pre></td>
```

Comment:
From this row down, there's no description cell which messes up the table format.

![2019-10-03_162934_064269253](https://user-images.githubusercontent.com/3689782/66161751-0716d680-e5fb-11e9-9692-0f3207085de3.png)

Reply:
oop. fixing